octodns/provider/ovh.py

```diff
@@ -323,7 +323,7 @@ def _is_valid_dkim(self, value):
                           'n': lambda _: True,
                           'g': lambda _: True}
 
-        splitted = value.split('\\;')
+        splitted = list(filter(None, value.split('\\;')))
```

Comment:
The python 3 porting guides encourage comprehensions rather than `list(filter(` for these cases. I believe that'd be something like the following here:

```suggestion
        splitted = [v for v in value.split('\\;') if v]
```

Reply:
Clearer indeed. Thanks for the help.